app/views/open_studios_subdomain/_container_header.slim

```diff
@@ -0,0 +1,6 @@
+.container-header
+  a.container-header__brand href=root_path title="Mission Artists"
+    .container-header__logo title="Mission Artists" class="#{Date.today.month == 6 ? 'sidebar-logo--pride' : ''}"
+    h2.container-header__mau-title Mission Artists
```

Comment:
Let's take this out on mobile.  I think you can do something like `.pure-u-sm-hidden` and you might get it for (almost) free.

Also - I love that you maintained the pride logo swap!